### PR TITLE
Adopt switch tab for pulsar-admin, REST API, and Java

### PR DESCRIPTION
**Pull request by sijia-w**
### Motivation

Adopt switch tab for pulsar-admin, REST API, and Java.

### Modification

Add switch tabs in Admin API docs.

### CODE REVIEW

**Review by Jennifer88huang — APPROVED** [review]

File: site2/docs/admin-api-tenants.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ my-tenant-2
 
 ### Create
 
-#### pulsar-admin
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (Jennifer88huang) [inline site2/docs/admin-api-tenants.md:33]: For this file, only `pulsar-admin` is listed, so it's not necessary to adopt switch tab.
You can try to refine the language from technical writing part to make it more clear.

**Comment by sijia-w** [comment]
@Jennifer88huang Updated. PTAL. Thanks.

**Comment by sijia-w** [comment]
@zymap Can you help review the added part in admin-api-tenants? Thank you.

**Review by wolfstudy — APPROVED** [review]

File: site2/docs/admin-api-tenants.md [hunk 2]
```diff
@@ -67,19 +93,64 @@ $ pulsar-admin tenants get my-tenant
...
+<!--REST API-->
 
-#### pulsar-admin
+{@inject: endpoint|DELETE|/admin/v2/tenants/:cluster|operation/updateTenant}
+
+<!--JAVA-->
+
+```java
+
+admin.tenants().updateTenant(tenantName);
```
Comment (zymap) [inline site2/docs/admin-api-tenants.md:153]:
```suggestion
admin.tenants().updateTenant(tenantName, tenantInfo);
```

File: site2/docs/admin-api-tenants.md [hunk 3]
```diff
@@ -67,19 +93,64 @@ $ pulsar-admin tenants get my-tenant
...
 }
 ```
+<!--REST API-->
+
+{@inject: endpoint|GET|/admin/v2/tenants/:cluster|operation/getTenant}
+
+<!--JAVA-->
+
+```java
+admin.tenants().getTenant(tenantName);
```
Comment (zymap) [inline site2/docs/admin-api-tenants.md:103]:
```suggestion
admin.tenants().getTenantInfo(tenantName);
```

File: site2/docs/admin-api-tenants.md [hunk 4]
```diff
@@ -47,12 +59,26 @@ $ pulsar-admin tenants create my-tenant \
...
   -r role1
 ```
+<!--REST API-->
+
+{@inject: endpoint|POST|/admin/v2/functions/{tenant}
+
+<!--JAVA-->
+
+```java
+admin.tenants().createTenant(tenantName);
```
Comment (zymap) [inline site2/docs/admin-api-tenants.md:69]:
```suggestion
admin.tenants().createTenant(tenantName, tenantInfo);
```

**Review by zymap — APPROVED** [review]
Overall looks good to me. Just needs some minor changes.

**Review by sijie — APPROVED** [review]

**Pull request merged**